modules/virt-enabling-descheduler-evictions.adoc

```diff
@@ -45,4 +45,19 @@ spec:
 ----
 <1> By default, the descheduler does not evict pods. To evict pods, set `mode` to `Automatic`.
 
+. Enable `devEnableEvictionsInBackground` to improve VM eviction stability during live migrations. This option allows evictions to occur in the background, mitigating issues with oscillatory behavior during live migration:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: KubeDescheduler
+metadata:
+  name: cluster
+  namespace: openshift-kube-descheduler-operator
+spec:
+  devEnableEvictionsInBackground: true
```

Comment:
Would it make more sense to add this option to step 2's example? So that they set all the necessary settings for the KubeDescheduler object in one step?